tests/CMakeLists.txt

```diff
@@ -4,6 +4,9 @@ set_property(TARGET mt_kahypar_tests PROPERTY CXX_STANDARD 17)
 set_property(TARGET mt_kahypar_tests PROPERTY CXX_STANDARD_REQUIRED ON)
 target_link_libraries(mt_kahypar_tests ${Boost_LIBRARIES} TBB::tbb TBB::tbbmalloc_proxy)
 
+# suppress warnings generated by TYPED_TEST_SUITE macro from gtest
+target_compile_options(mt_kahypar_tests PRIVATE "-Wno-gnu-zero-variadic-macro-arguments")
```

Comment:
How spammy are these warnings? If it's just one, it may be fine to keep it. Up to you.